jet_tracking/jet_tracking_cal/submit_jt_arp.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source /reg/g/psdm/sw/conda1/manage/bin/psconda.sh
+ABS_PATH=/cds/home/a/aegger/jet_tracking/jet_tracking/jet_tracking_cal
+sbatch --nodes=2 --time=5 $ABS_PATH/jt_cal.py "$@"
```

Comment:
Where is `sbatch` meant to come from? I don't see in on my path after sourcing `/reg/g/psdm/sw/conda1/manage/bin/psconda.sh`

Reply:
sbatch is recognized on psana for slurm batch jobs.